### ISSUE TITLE
improve HTTP Communication

### ISSUE THREAD

**Issue by lnetwalker** [lnetwalker/openMSR#70]
currently we use a socket only once, so we have lot's of sockets in TIME_WAIT state. This can be a bottlaneck. So I want to use HTTP 1.1 and Keep-alive to better user the sockets

**Comment by lnetwalker** [comment]
perhaps its an option to replace the webserver
https://wiki.freepascal.org/Light_Web_Server

**Comment by lnetwalker** [comment]
it would be also nice to have https and use encrypted communication

**Comment by lnetwalker** [comment]
perhaps it is an option to add a UDP server to DeviceServer and do PhysMach Communication over UDP!
